pandas/tseries/frequencies.py

```diff
@@ -278,7 +278,8 @@ def get_period_alias(offset_str):
 
 def to_offset(freqstr):
     """
-    Return DateOffset object from string representation
+    Return DateOffset object from string representation or
+    datetime.timedelta or pandas.tslib.Timedelta objects
```

Comment:
pandas.tslib.Timedelta -> pandas.Timedelta (that is how it is exposed to the users)